src/chrome/content/rules/Nurago.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="Nurago">
+    <target host="sensic.net" />
+    <target host="*.sensic.net" />
+
+    <rule from="^http://(mco-pb|www)\.sensic\.net/"
+        to="https://$1.sensic.net/" />
```

Comment:
Also add

    <rule from="^http://sensic\.net/"
            to="https://sensic.net/" />